### ISSUE TITLE
Sense of impending doom

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6993]
**Preferred term label:**
Sense of impending doom

**Synonyms**
Sense of doom

**Definition (free text, please give PubMed ID)**
A feeling that something life-threatening or tragic is about to occur.
PMID:18577027

**Parent term (use hpo.jax.org/app)**
Abnormal fear/anxiety-related behavior HP:0100852

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Long COVID

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
